central/auth/datastore/datastore_impl_test.go

```diff
@@ -193,7 +316,7 @@ func (s *datastorePostgresTestSuite) addRoles() {
 	}))
 
 	for _, role := range testRoles.AsSlice() {
-		s.Require().NoError(s.roleDataStore.AddRole(s.ctx, &storage.Role{
+		s.Require().NoError(roleStore.Upsert(s.ctx, &storage.Role{
```

Comment:
This was changed because using the data store raised an error because `Configuration Controller` is a default role and you can't create "custom" roles with the same name as a default role, but the default roles are not inserted into the DB like the data store assumes!